src/components/structures/LoggedInView.js

```diff
@@ -245,6 +263,9 @@ export default React.createClass({
         if (topBar) {
             bodyClasses += ' mx_MatrixChat_toolbarShowing';
         }
+        if (this.state.useCompactLayout) {
+            bodyClasses += ' mx_MatrixChat_useCompactLayout';
+        }
```

Comment:
Since it is now getting a little more complex any chance you could pleez incorporate classnames
e.g like here https://github.com/matrix-org/matrix-react-sdk/blob/443ab1add73390176478fc7ecd1a334aa157e833/src/components/views/rooms/MemberInfo.js#L637-L637